### ISSUE TITLE
Issue by using multiply robots

### ISSUE THREAD

**Issue by Andi1927** [cactusss/nl.onzewifi.DeebotT9#2]
First of all great app and working fine with one robot.
I have two one for each floor, one T8 and one T9+.
If I start the T9 I cant access anymore the T8 and vice versa. I had to restart the app and then I can access the other one. Do you have any Idea ?

**Comment by cactusss** [comment]
No, I don't. I unfortunaly have only one so Deebot, so I don't know how to reproduce or debug your problem. I forked the app Olivier Messiant created and build on from that, but never reaaly looked at the driver part.

What you could do install his app (https://homey.app/en-gb/app/com.messiant.ecovacs/Ecovacs/) and see if that app has the same problem. If it doesn't it must be caused by my code, and I will have a look if I can figure out what goes wrong. If that app has the same problem I'm afraid I really can't do anything about it since I can't test it with two Deebots.

Another possibility is to change the app name (change nl.onzewifi.DeebotT9 in the app.json) and run two instances of the app, one fot the T9 and one for the T8, but again, I have no possibility to test if that will work either.

Sorry :)

**Comment by cactusss** [comment]
Also, send me a crash report please. Mayby I can see what happens in there.

**Comment by Andi1927** [comment]
I came from the other app and it had the same problem.
What you mean with crash report ? There is no crash only it doesnt work.
Can you explain this with the second instance more in details ?

**Comment by cactusss** [comment]
For this you need to know how to do a CLI install. The Homey forum can help you out if you have any questions.

What you need to do is doenload the source of the app (https://github.com/cactusss/nl.onzewifi.DeebotT9/archive/refs/heads/1.2.5.zip) and extract the zip file somewhere. In the .homeycompose folder you'll find a file called app.json. Open this with atext editor, and change the ID ("nl.onzewifi.deebott9") and the name ("< Ecovacs Deebot >").

After doing a CLI install on your homey you'll have (beside the current app) a new app with the name you gave it. I don't guarantee it will work, but it's worth a try.
